src/azure-cli/azure/cli/command_modules/vm/tests/latest/recordings/test_gallery_e2e.yaml

```diff
@@ -3340,7 +3340,7 @@ interactions:
       User-Agent:
       - azsdk-python-keyvault-keys/4.5.0b6 Python/3.8.10 (Windows-10-10.0.22000-SP0)
     method: POST
-    uri: https://vault-000002.vault.azure.net/keys/key-000004/create?api-version=7.3-preview
+    uri: https://vault-000002.vault.azure.net/keys/key-000004/create?api-version=7.3
```

Comment:
Just out of curiosity, I would like to ask why the preview api-version is used in the case of GA